docs/src/pages/docs/crocks/Maybe.md

```diff
@@ -1445,6 +1445,42 @@ Just('so good')
 //=> Just "so good"
 ```
 
+#### find
+
+`crocks/Maybe/find`
+
+```haskell
+find :: Foldable f => ((a -> Boolean) | Pred) -> f a -> Maybe a
+```
+Used to take a foldable and a function or predicate and searchs through it
```

Comment:
Also, we have a link to the `Pred` docs at the bottom, so that refrerence to `Pred` should link to:

<code>[&#96;Pred&#96;][pred]</code>

But only in this document not the README in Maybe